osmosis/assetlist.json

```diff
@@ -11568,9 +11568,9 @@
       ],
       "type_asset": "ics20",
       "base": "ibc/1B708808D372E959CD4839C594960309283424C775F4A038AAEBE7F83A988477",
-      "name": "Quasar",
-      "display": "qsr",
-      "symbol": "QSR",
+      "name": "Quasar (legacy)",
+      "display": "qsr.legacy",
```

Comment:
This 'display' unit is not defined anywhere, and therefore the exponent is unknown. I recommend changing this back to what it was.

Reply:
Fixed the "display" unit in PR as per feedback